com.playeveryware.eos/Runtime/Core/EOSManager.cs

```diff
@@ -1061,83 +1061,45 @@ public void ConnectLinkExternalAccountWithContinuanceToken(ContinuanceToken toke
             /// Also contains the information needed to set ProductUserId.
             /// <see cref="s_localProductUserId"/>
             /// </param>
-            public async void StartConnectLoginWithEpicAccount(EpicAccountId epicAccountId,
-                OnConnectLoginCallback onConnectLoginCallback)
+            public async void StartConnectLoginWithEpicAccount(EpicAccountId epicAccountId, OnConnectLoginCallback onConnectLoginCallback)
             {
-                var EOSAuthInterface = GetEOSPlatformInterface().GetAuthInterface();
-                var copyUserTokenOptions = new CopyUserAuthTokenOptions();
-                var result =
-                    EOSAuthInterface.CopyUserAuthToken(ref copyUserTokenOptions, epicAccountId, out Token? authToken);
-                var connectLoginOptions = new Epic.OnlineServices.Connect.LoginOptions();
+                var authInterface = GetEOSPlatformInterface().GetAuthInterface();
 
-                if (result == Result.NotFound)
+                var idOpts = new Epic.OnlineServices.Auth.CopyIdTokenOptions
                 {
-                    Log("No User Auth tokens found to login");
-                    if (onConnectLoginCallback != null)
-                    {
-                        var dummyLoginCallbackInfo = new Epic.OnlineServices.Connect.LoginCallbackInfo();
-                        dummyLoginCallbackInfo.ResultCode = Result.ConnectAuthExpired;
-                        onConnectLoginCallback(dummyLoginCallbackInfo);
-                    }
-
-                    return;
-                }
+                    AccountId = epicAccountId
+                };
 
-                Log($"CopyUserAuthToken result code: {result}");
+                var result = authInterface.CopyIdToken(ref idOpts, out Epic.OnlineServices.Auth.IdToken? idToken);
 
-                if (!authToken.HasValue)
+                if (result != Result.Success || !idToken.HasValue)
                 {
-                    Log("authToken was not found, unable to login");
-
-                    var dummyLoginCallbackInfo = new Epic.OnlineServices.Connect.LoginCallbackInfo();
-                    dummyLoginCallbackInfo.ResultCode = Result.InvalidAuth;
-                    onConnectLoginCallback(dummyLoginCallbackInfo);
-
+                    Debug.LogError("[EOS] CopyIdToken failed" + result);
```

Comment:
As below, minor change but having this additional context is handy
```suggestion
                    Debug.LogError($"{nameof(EOSManager)} {nameof(StartConnectLoginWithEpicAccount)}: CopyIdToken failed with result: {result}");
```